docs/connecting.txt

```diff
@@ -51,11 +51,32 @@ the following two formats are both allowed::
     jdbc:crate://localhost:5432/
 
 
-In order to connect to multiple servers multiple ``<host>:<psql-port>``
-pairs can be specified by delimit them using a comma::
+To support connection fail-over, multiple pairs of <host>:<psql-port> dilimited
+by a comma can be specified::
 
     crate://host1.example.com:5432,host2.example.com:5432/
 
+The driver will try to connect to the hosts in the order defined by the comma
+separated list until the first successfull connection is established.
+
+.. note::
+
+    If you have multiple client apps/threads connecting to the CrateDB cluster,
+    it is adviseable to use shuffle the order of those pairs in each client 
+    app/thread, which will allow a convenient connection load balancing accross
```

Comment:
"in each instance"

"as this will provide a cheap form of load balancing. The goal being that each client or thread is connecting to a different node by default, and then moving on to a different node if that node is not available, and so on. In aggregate, this should spread load fairly evenly.